screengrab/lib/screengrab/runner.rb

```diff
@@ -146,8 +149,11 @@ def determine_external_screenshots_path(device_serial)
       File.join(device_ext_storage, @config[:app_package_name], 'screengrab')
     end
 
-    def determine_internal_screenshots_path
-      "/data/data/#{@config[:app_package_name]}/app_screengrab"
+    def determine_internal_screenshots_paths
+      [
+        "/data/data/#{@config[:app_package_name]}/app_screengrab",
+        "/data/user/0/#{@config[:app_package_name]}/files/#{@config[:app_package_name]}/screengrab//en-US/images/screenshots"
```

Comment:
probably not an issue, but there's two slashes `//en-US`

Reply:
Whooooopppss